CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ Changelog for the runtimes governed by the Polkadot Fellowship.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [1.3.4] 31.10.2024
```

Comment:
```suggestion
## [1.3.4] 01.11.2024
```
WDYT about today? Or are there outstanding things to fix?

Reply:
I was just a bit paranoid about the inflation change since its so significant. Checked it twice now again locally, should be good.

Reply:
yea fair enough!